charts/datasaur/templates/config/datasaur-env-configMap.yaml

```diff
@@ -33,6 +33,7 @@ data:
   ENABLE_NEW_ANALYTICS: "true"
   ENABLE_CUSTOM_REPORT_BUILDER: "true"
   METABASE_BASE_URL: http://{{ .Release.Name }}-metabase
+  METABASE_SECRET_KEY: c4a855d8bf60e61e86a689f3d1aef36b0f4bbff9097945f18395c1db4ebda627
```

Comment:
is this a real key? kalau iya, kudu dihapus dari private repo ini, suruh diinput dari values.yaml nya customer

@billy-editiano